domain/schema/model.go

```diff
@@ -147,7 +147,7 @@ func ModelDDL() *schema.Schema {
 		triggers.ChangeLogTriggersForRelationStatus("relation_uuid",
 			tableRelationStatus),
 		triggers.ChangeLogTriggersForRelationUnit("unit_uuid", tableRelationUnit),
-		triggers.ChangeLogTriggersForIpAddress("uuid", tableIpAddress),
+		triggers.ChangeLogTriggersForIpAddress("net_node_uuid", tableIpAddress),
```

Comment:
Could this break other watchers?

Reply:
No, this watcher is only used in this PR and in any case watching the ip_address is supposed to have a way of filtering, i.e. net_node_uuid.

Reply:
I would put a comment noting the deviation from standard PK change identifier, and why we're doing it. I.e. that we care about nodes changing their addresses, rather than specific addresses changing.